var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -161,7 +161,10 @@ def cmake_args(self):
                     self.spec['geos'].prefix.bin,
                     '-DGSL_CONFIG_PREFER_PATH=' + self.spec['gsl'].prefix.bin,
                     '-DPOSTGRES_CONFIG_PREFER_PATH=' +
-                    self.spec['postgresql'].prefix.bin
+                    self.spec['postgresql'].prefix.bin,
+                    '-DPROJ_INCLUDE_DIR=' + self.spec['proj'].prefix.include,
+                    '-DSQLITE3_LIBRARY='
+                    + join_path(self.spec['sqlite'].prefix.lib, 'libsqlite3.so')
```

Comment:
This won't work if the library is in lib64 or you're on macOS, use this instead:
```suggestion
                    + self.spec['sqlite'].libs[0]
```

Reply:
great!